### DISCUSSION TITLE
图像比对

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#685] [Q&A]
现在没有   /api/v1/recognition/faces/{imagesId}/verify   通过上传的图像加imagesId进行对比的API没有了嘛？

**Comment by pospielov** [comment]
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#verify-faces-from-a-given-image
Using this endpoint you can compare your image with the image_id saved in the database.

**Reply by yang1158280920** [reply]
Can the images uploaded by this API be in Base64 format?

**Reply by pospielov** [reply]
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#verify-faces-from-a-given-image-base64
yes, you just need to use application/json, like {"file": "<base64_value>"}

**Reply by yang1158280920** [reply]
Does the API in this endpoint all support the Base64 format?

**Reply by pospielov** [reply]
Every REST Endpoint that works with images support base64 format

**Reply by yang1158280920** [reply]
thanks
